web/src/flow/stages/captcha/CaptchaStage.ts

```diff
@@ -110,9 +115,9 @@ export class CaptchaStage extends BaseStage<CaptchaChallenge, CaptchaChallengeRe
         if (!Object.hasOwn(window, "turnstile")) {
             return false;
         }
-        window.turnstile.render(container, {
+        (window as unknown as TurnstileWindow).turnstile.render(container, {
             sitekey: this.challenge.siteKey,
-            size: "invisible",
+            size: "invisible" as TurnstileOptions["size"],
```

Comment:
@BeryJu Could you please affirm this?  The `as TurnstilesOptions["size"]` clause was necessary here because "invisible" is not a valid setting according to the options in the provided type file.  If `invisible` is valid because you've made it so, let me know and I'll a comment to that effect.

Reply:
I don't think thats valid (at least not anymore), so we can probably remove it